src/domain/enrolment/__tests__/utils.test.ts

```diff
@@ -90,8 +85,9 @@ describe('getEnrolmentPayload function', () => {
       dateOfBirth = '10.10.1999',
       email = 'Email',
       extraInfo = 'Extra info',
+      firstName = 'First name',
+      lastName = 'First name',
```

Comment:
Should probably be lastName = 'Last name',

Reply:
Oh, sure it should. Good catch.

Reply:
Fixed in https://github.com/City-of-Helsinki/linkedregistrations-ui/pull/60/commits/4e199294470841d168d96c5821740600f23d5555